### PR TITLE
chore: move batch.js file from bloomstack_core into erpnext

### PR DESCRIPTION
**Pull request by imdadhussain**
**Task ID**: https://bloomstack.com/desk#Form/Task/TASK-2021-00277

**Description**: 
Delete: batch.js file created in erpnext .
Move: batch.js file from bloomstack_core and utils function.

**Tested**: YES 

**Gif Added**: 
batch

**Dependency PR**: https://github.com/Bloomstack/erpnext/pull/1643

**Bloomstack Core PR**: https://github.com/Bloomstack/bloomstack_core/pull/668

### CODE REVIEW

File: erpnext/stock/doctype/batch/batch.js [hunk 1]
```diff
@@ -46,6 +46,28 @@ frappe.ui.form.on('Batch', {
...
+				frappe.prompt({
+					label: "Target Warehouse",
+					fieldname: "warehouse",
+					fieldtype: "Link",
+					options: "Warehouse",
+					reqd: 1
+				},
+				function (data) {
+					frappe.model.open_mapped_doc({
+						method: "bloomstack_core.utils.move_expired_batches",
```
Comment (vishdha) [inline erpnext/stock/doctype/batch/batch.js]: @imdadhussain  Path should be correct

**Review by vishdha — CHANGES_REQUESTED** [review]

File: erpnext/utils.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 import frappe
 from frappe.utils import get_url
 
+
```
Comment (sahil28297) [inline erpnext/utils.py]:
```suggestion
```

**Review by sahil28297 — CHANGES_REQUESTED** [review]

File: erpnext/utils.py [hunk 3]
```diff
@@ -52,4 +52,4 @@ def create_authorization_request(dt, dn, contact_email, contact_name):
 	new_authorization_request.linked_docname = dn
 	new_authorization_request.authorizer_email = contact_email
 	new_authorization_request.authorizer_name = contact_name
-	new_authorization_request.save()
\ No newline at end of file
+	new_authorization_request.save()
```
Comment (sahil28297) [inline erpnext/utils.py:55]:
```suggestion
	new_authorization_request.save()
```

**Review by sahil28297 — CHANGES_REQUESTED** [review]

File: erpnext/utils.py (hunk 3 above)
Comment (sahil28297) [inline erpnext/utils.py:55]:
```suggestion
	new_authorization_request.save()

```

**Review by sahil28297 — CHANGES_REQUESTED** [review]

File: erpnext/utils.py [hunk 4]
```diff
@@ -52,4 +52,5 @@ def create_authorization_request(dt, dn, contact_email, contact_name):
...
```
Comment (sahil28297) [inline erpnext/utils.py]: revert this file change

**Review by sahil28297 — CHANGES_REQUESTED** [review]

File: erpnext/stock/doctype/batch/batch.js [hunk 5]
```diff
@@ -45,7 +45,29 @@ frappe.ui.form.on('Batch', {
 			}, __("Create"));
 			this.frm.page.set_inner_btn_group_as_primary(__('Create'));
 		}
+
```
Comment (sahil28297) [inline erpnext/stock/doctype/batch/batch.js]:
```suggestion
```

File: erpnext/stock/doctype/batch/batch.js [hunk 6]
```diff
@@ -45,7 +45,29 @@ frappe.ui.form.on('Batch', {
...
+						frm: frm,
+						args: {
+							warehouse: data.warehouse
+						}
+					});
+				}, __("Select Warehouse"), __("Move"));
+			});
+		}
 	},
+
```
Comment (sahil28297) [inline erpnext/stock/doctype/batch/batch.js]:
```suggestion
```

**Review by sahil28297 — CHANGES_REQUESTED** [review]

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**